.gitignore

```diff
@@ -0,0 +1,4 @@
+dist
+target
+node_modules
+.idea/
```

Comment:
tip: I suggest you add this line to your [global gitignore](https://help.github.com/articles/ignoring-files/), then you'll never need it again